package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "core-standards",
+  "version": "1.0.0",
```

Comment:
The version number in package.json should be identical to the version in the main plugin PHP file.